contrib/ci/Dockerfile-fedora.in

```diff
@@ -6,6 +6,7 @@ ENV LC_ALL en_US.UTF-8
 RUN echo fubar > /etc/machine-id
 RUN dnf -y update
 RUN echo fubar > /etc/machine-id
+RUN dnf install -y ShellCheck
```

Comment:
is the package name case sensitive?

Reply:
Installing it here is going to be problematic in terms of our containers are only generated once per day.  At least for now, I would suggest also install it in `contrib/ci/fedora.sh`, and once we have updated base containers we drop it form the CI part.

Reply:
> is the package name case sensitive?

Yup.

> I would suggest also install it in contrib/ci/fedora.sh

Makes sense, thanks.